### PR TITLE
Remove "calc and eval going away" warning banners

### PR DESCRIPTION
**Pull request by bencodeorg**
Removes banners that warns users with existing calc and eval projects that their projects are going away. These banners will no longer appear on the /home and /projects URLs for project owners, and on share URLs for these project types for all users.

Basically a revert of [this PR](https://github.com/code-dot-org/code-dot-org/pull/52722), although I had to do it manually because github said it couldn't revert automagically :/

Note we still have a banner that appears if you directly navigate to a calc or eval project as the project owner warning you that the lab is going away.

## Links

- jira ticket: [LABS-258](https://codedotorg.atlassian.net/browse/LABS-258)

## Testing story

Tested manually on a teacher and student account that the banner disappeared from /home as well as from /projects. Also tested manually that a share URL no longer showed the banner.

Also did a bit of skimming to make sure no new code had used any code introduced in this PR and didn't see anything.

### CODE REVIEW

File: dashboard/lib/projects_list.rb [hunk 1]
```diff
@@ -41,16 +41,6 @@ def fetch_personal_projects(user_id)
...
 
-    def user_has_project_type(user_id, types)
-      storage_id = storage_id_for_user_id(user_id)
-      Projects.new(storage_id).get_active_projects.each do |project|
-        if types.include?(project[:project_type])
-          return true
-        end
-      end
-      false
-    end
```
Comment (breville) [inline dashboard/lib/projects_list.rb]: I wonder if this could be useful for someone else in the future?

**Review by breville — APPROVED** [review]
Looks good.  Thanks for checking that later code didn't take dependencies on any of this!

File: dashboard/lib/projects_list.rb (hunk 1 above)
Comment (fisher-alice) [reply dashboard/lib/projects_list.rb]: +1 in light of plans to eventually deprecate labs like playlab and weblab.

Comment (bencodeorg) [reply dashboard/lib/projects_list.rb]: Sure, I can keep it around

**Pull request merged**